src/layouts/buttonsContainer/styles.css

```diff
@@ -0,0 +1,17 @@
+
+.container{
+    grid-area: 'buttons';
```

Comment:
Should be without ''. It won't work as expected because the container is inside the section with the class `index-page__container` instead of directly inside `div#main-grid-container` that defines the `grid-template-areas`.